airflow/www/app.py

```diff
@@ -1055,7 +1055,7 @@ def tree(self):
         num_runs = int(num_runs) if num_runs else 25
 
         if not base_date:
-            base_date = dag.latest_execution_date or datetime.now()
+            base_date = (dag.latest_execution_date + 2 * dag.schedule_interval) or datetime.now()
```

Comment:
In the case of a new DAG dag.latest_execution_date will be `None` and this will raise an error, you'll need an `if` here...